src/components/Share/style.scss

```diff
@@ -0,0 +1,30 @@
+.share {
```

Comment:
sorry, i did not get " css-module", i am thinking that we are using SCSS that support nesting, right?

http://sass-lang.com/guide#topic-3